sw/device/silicon_creator/rom/keys/README.md

```diff
@@ -7,9 +7,9 @@ title: "Signing Keys"
 [Open Titan Tool](ott) is the recommended tool to generate RSA keys to use in
 the `ROM`.
 
-Inspecting a key (show the public information of a key):
+- Inspecting a key (show the public information of a key):
 ```
-$ /path/to/opentitantool rsa key show <basename>.der
+$ bazel run //sw/host/opentitantool -- rsa key show $PWD/<basename>.der
```

Comment:
The first commit is also changing this file, could you squash them or move this change to the first commit?

Reply:
Yes indeed, I  have reworked that.